magicctapipe/io/io.py

```diff
@@ -66,6 +58,55 @@
 DEAD_TIME_LST = 7.6 * u.us
 DEAD_TIME_MAGIC = 26 * u.us
 
+def telescope_combinations(config):
```

Comment:
I think it would be good to add in tests/test_io.py a new unit test for this function (just passing configs of MAGIC + LST1 and 4 LSTs and compare the result of the function with hardcoded answer

Reply:
Done

Reply:
thanks!